src/Common/src/Interop/Windows/NtDll/Interop.FILE_FULL_EA_INFORMATION.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class NtDll
+    {
+        /// <summary>
+        /// <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/content/wdm/ns-wdm-_file_full_ea_information">FILE_FULL_EA_INFORMATION</a> structure.
+        /// Provides extended attribute (EA) information. This structure is used primarily by network drivers.
+        /// </summary>
+        [StructLayoutAttribute(LayoutKind.Sequential)]
```

Comment:
Nit: Structs are sequential by default. We will add them anyway sometimes to avoid warnings about naming if they pop up.